Sources/Core/Renderer/RenderCoordinator.swift

```diff
@@ -53,19 +102,18 @@ public final class RenderCoordinator: NSObject, MTKViewDelegate {
 
   /// The longest a frame has taken to encode so far.
   private var longestFrame: Double = 0
-
+  
```

Comment:
Just a minor thing, but I usually avoid trailing whitespace (my editor removes it but I get that not all editors do that)

Reply:
Whoops that might have been me on Xcode

Reply:
it has been removed